### PR TITLE
refactor RecipeResult

### PR DESCRIPTION
**Pull request by subinakallyani**

### CODE REVIEW

File: src/js/controller.js [hunk 1]
```diff
@@ -215,33 +218,34 @@ searchContainer.addEventListener('submit', async e => {
...
+  //               </div>
+  //             </div>
+  //           </a>
+  //         </li>`;
+  //   })
+  //   .join('');
+  // //console.log(searchResult);
+  // searchResults.innerHTML = '';
+  // searchResults.insertAdjacentHTML('afterbegin', searchResult);
+  // searchBox.value = '';
```
Comment (pranavmappoli) [inline src/js/controller.js:248]: Why dont you remove this lines ?

File: src/model.js [hunk 2]
```diff
@@ -59,4 +59,4 @@ export async function searchResult(search) {
   }
   //throw err;
 }
-searchResult('pizza');
+//searchResult('pizza');
```
Comment (pranavmappoli) [inline src/model.js]: remove this

File: src/views/results.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      console.log(item, 'hhhh');
+      return ` 
+<li class="preview">
+        <a class="preview__link preview__link--active" href="#${item.id}">
+          <figure class="preview__fig">
+            <img src="${item.image}" alt="Test" />
+          </figure>
+          <div class="preview__data">
+            <h4 class="preview__title">${item.title} ...</h4>
+            <p class="preview__publisher">${item.publisher}</p>
```
Comment (pranavmappoli) [inline src/views/results.js:16]: rename the file to receipeResults something

**Review by pranavmappoli — CHANGES_REQUESTED** [review]

File: src/model.js (hunk 2 above)
Comment (subinakallyani) [reply src/model.js]: I have removed

**Pull request merged**